### DISCUSSION TITLE
Renaming entity names from parent device

### DISCUSSION THREAD

**Discussion by stasjminister** [zwave-js/zwave-js-ui#2218] [Ideas]
When adding a new device in zwavejs2mqtt or zwavejs. I have to manually change entity names after the device name and location is set. Please add an option to rename child entities like you are able to in zigbee2mqtt.

**Comment by scstraus** [comment]
Very much needed. All the entity renaming when adding new devices is a massive waste of time since I'm just naming them all the same thing with the suffix that's already on the entity.

**Reply by robertsLando** [reply]
@stasjminister If you are using hass integration I have no control on child entities names. 

cc @raman325

**Reply by stasjminister** [reply]
Yes this is in the hass integration. Thanks for forwarding :)
